modes/assets/semeru/start-quarkus.sh

```diff
@@ -11,7 +11,7 @@ else
     mkdir cr
     $JAVA_HOME/bin/java -XX:CRaCCheckpointTo=cr -Dopenj9.internal.criu.unprivilegedMode=true ${JAVA_OPTS} ${JAVA_OPTS_APPEND} -jar ${JAVA_APP_JAR} 1>out 2>err </dev/null &
 
-    sleep 5
+    sleep 10
```

Comment:
I had to increase this sleep to 10s because I noticed that sometimes it takes more than 5s to startup.. therefore the server was not available when trying to hit the endpoint